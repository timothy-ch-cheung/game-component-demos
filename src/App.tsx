import React from "react";
import { Background } from "./components/Background";
import { NavBar } from "./components/NavBar";

function App() {
  return (
    <Background>
      <NavBar />
    </Background>
  );
}

export default App;
