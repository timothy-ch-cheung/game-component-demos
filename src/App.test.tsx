import React from 'react';
import { render } from '@testing-library/react';
import App from './App';

test('Renders App', () => {
  const {container} = render(<App />);
  expect(container).toMatchSnapshot();
});
