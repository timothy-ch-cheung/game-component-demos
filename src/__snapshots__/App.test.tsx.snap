// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Renders App 1`] = `
<div>
  <div>
    <div
      class="css-14fp8fi"
    />
    <div
      class="css-19qgtyd"
    />
    <div
      class="css-ajann8"
    />
    <div>
      <header
        class="MuiPaper-root MuiPaper-elevation MuiPaper-elevation4 MuiAppBar-root MuiAppBar-colorPrimary MuiAppBar-positionStatic css-hip9hq-MuiPaper-root-MuiAppBar-root"
        style="background: rgb(79, 79, 75);"
      >
        <div
          class="MuiToolbar-root MuiToolbar-gutters MuiToolbar-regular css-hyum1k-MuiToolbar-root"
        >
          <button
            class="MuiButtonBase-root MuiButton-root MuiButton-text MuiButton-textPrimary MuiButton-sizeMedium MuiButton-textSizeMedium MuiButton-root MuiButton-text MuiButton-textPrimary MuiButton-sizeMedium MuiButton-textSizeMedium css-1csl68j-MuiButtonBase-root-MuiButton-root"
            tabindex="0"
            type="button"
          >
            Home
            <span
              class="MuiTouchRipple-root css-8je8zh-MuiTouchRipple-root"
            />
          </button>
          <button
            class="MuiButtonBase-root MuiButton-root MuiButton-text MuiButton-textPrimary MuiButton-sizeMedium MuiButton-textSizeMedium MuiButton-root MuiButton-text MuiButton-textPrimary MuiButton-sizeMedium MuiButton-textSizeMedium css-qx4n15-MuiButtonBase-root-MuiButton-root"
            tabindex="0"
            type="button"
          >
            Game Demos
            <span
              class="MuiTouchRipple-root css-8je8zh-MuiTouchRipple-root"
            />
          </button>
        </div>
      </header>
    </div>
  </div>
</div>
`;
